tests/integration/components/ak-tabs/index-test.js

```diff
@@ -0,0 +1,174 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, findAll, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | ak-tabs', function (hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(async function () {
+    this.setProperties({
+      buttonVariant: true,
+      activeTab: 1,
+
+      tabItems: [
+        {
+          id: 1,
+          hidden: false,
+          badgeCount: 9,
+          hasBadge: true,
+          label: 'Tab One',
+          iconName: 'assignment',
+        },
+        {
+          id: 2,
+          hidden: false,
+          badgeCount: 100,
+          hasBadge: true,
+          label: 'Tab Two',
+          iconName: 'people',
+        },
+      ],
+
+      onTabClick: (tabId) => {
+        this.set('activeTab', tabId);
+        this.currentTabDetails = this.tabItems.find(
+          (tab) => tab.id === this.activeTab
+        );
+      },
+    });
+  });
+
+  test('it renders ak-tabs with one tab item', async function (assert) {
+    await render(hbs`
+      <AkTabs as |Akt|>
+        <Akt.tabItem
+          @buttonVariant={{true}}
+          @id="tab-item"
+          @hasBadge={{true}}
+          @badgeCount={{9}}
+          @isActive={{true}}
+        >
+          <:tabIcon>
+            <span data-test-tab-icon>{{this.iconName}}</span>
+          </:tabIcon>
+
+          <:default>
+            Tab item
+          </:default>
+        </Akt.tabItem>
+      </AkTabs>
+    `);
+
+    assert.dom('[data-test-ak-tabs-container]').exists();
+    assert.dom('[data-test-ak-tab-list-wrapper]').exists();
+    assert.dom('[data-test-tab-icon]').exists();
+    assert.dom('[data-test-ak-tab-item]').exists().hasAnyText('Tab item');
```

Comment:
can we make this `hasText`

Reply:
Other text content are available in the tab item.

Reply:
An alternatives would be `containsText` but the `hasAnyText` is just fine also.